build/secondary/third_party/vulkan_validation_layers/BUILD.gn

```diff
@@ -0,0 +1,490 @@
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+################################################################################
+# This is a temporary secondary build file until the rtti issue is solved
+# in vulkan-deps
```

Comment:
Please file an issue tracking removal of this and link it here.

Reply:
Do you want one for flutter?  I'm not familiar with what bug tracker would be managing upstream issues.  There is a link to the fuchsia bug tracker in the code.

Reply:
done: https://github.com/flutter/flutter/issues/127624